ubiquity/dell-bootstrap.py

```diff
@@ -1360,6 +1348,14 @@ def install(self, target, progress, *args, **kwargs):
 
         #install dell-recovery only if there is an RP
         if rec_part:
+            #hide the recovery partition as default
+            try:
+                recovery = magic.find_factory_partition_stats()
```

Comment:
since the “dualrcvy” label put into recovery_common.py . If it is dual os，it will check only once when trying to find rec_part，it seems no worry to concern



-----Original Message-----
From: Mario Limonciello [notifications@github.com<mailto:notifications@github.com>]
Sent: Thursday, April 28, 2016 04:43 AM Malay Peninsula Standard Time
To: superm1/dell-recovery
Cc: Wang, Kevin Rustin; Author
Subject: Re: [superm1/dell-recovery] Trusty (#9)


In ubiquity/dell-bootstrap.py<https://github.com/superm1/dell-recovery/pull/9#discussion_r61332413>:


> @@ -1360,6 +1348,14 @@ def install(self, target, progress, *args, **kwargs):
>
>          #install dell-recovery only if there is an RP
>          if rec_part:
> +            #hide the recovery partition as default
> +            try:
> +                recovery = magic.find_factory_partition_stats()


I think you might need to change find_factory_partition_stats() because it might be mistakingly finding a Windows partition. It needs to know what to do in dual boot scenarios (probably needs to look for the label for dual boot recovery before other labels)

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/superm1/dell-recovery/pull/9/files/74a8d3201a2ffc479f5600118ca69c00066d2601#r61332413>